.github/workflows/python3.yaml

```diff
@@ -13,6 +13,9 @@ jobs:
       matrix:
         python-version: ["3.8", "3.9", "3.10"]
         twisted-version: [tx18, tx19, tx20, tx21]
+        include:
+          - python-version: "3.11"
```

Comment:
is there a reason these can't just be added to the lists above?

Reply:
I don't think Python 3.11 will work with older Twisted versions.